pages/quiz/helplines.tsx

```diff
@@ -65,27 +94,78 @@ const StyledLink = styled.a`
   }
 `
 
-export const IfYouNeedHelpPage = (): JSX.Element => {
+export const IfYouNeedHelpPage = ({
+  serviceData1,
+  serviceData2,
+}: HelplinePageProps): JSX.Element => {
+  const [selectedTriggerFeelings, setSelectedTriggerFeelings] = useState<
+    Array<string>
+  >([])
+
+  useEffect(() => {
+    const feelings = JSON.parse(
+      localStorage.getItem('quiz/howAreFeeling') || ''
+    )
+
+    const triggerFeelings = ['scared', 'unsafe', 'angry', 'ignored', 'sad']
+    const newTriggerFeelings = []
+
+    for (let i = 0; i < feelings.length; i++) {
+      if (triggerFeelings.find((element) => element === feelings[i])) {
+        newTriggerFeelings.push(feelings[i])
+        setSelectedTriggerFeelings(newTriggerFeelings)
+      }
+    }
+  }, [])
+
+  const makeFeelingsList = (array: string[]) => {
+    if (array.length === 1) return array[0]
+    const firstFeelings = array.slice(0, array.length - 1)
+    const lastFeeling = array[array.length - 1]
+    return firstFeelings.join(', ') + ' and ' + lastFeeling + '.'
+  }
+
   return (
     <Layout>
       <HeaderComponent title="Support in Lambeth" />
       <MainBody>
-        <Heading>Do you need help?</Heading>
-        <VerticalSpacing />
-        <Text>If you are in immediate danger, call 999.</Text>
+        <MainText>{`You've selected that you feel ${makeFeelingsList(
+          selectedTriggerFeelings
+        )}`}</MainText>
+        <VerticalSpacing size={2} />
+        <MainText>
+          Do you need help now or you are just having a bad day?
+        </MainText>
+        <VerticalSpacing size={2} />
+        <Link href="/quiz/what-are-your-interests" passHref>
```

Comment:
Not sure that this amount of text works here, maybe full width and give it some more padding or make it a link and keep text-decoration on?